packages/angularjs-extras/src/controllers/panel.js

```diff
@@ -44,8 +44,8 @@
 
       // Digest Collection Bindings
       $scope.collection = null
-      if ($element.attr('ng-collection')) {
-        $scope.$parent.$watch($element.attr('ng-collection'),
+      if ($element.attr('data-ng-collection')) {
```

Comment:
This should check for both, as technically both are valid possibilities.

Reply:
See other comment. I don't believe both are valid, only data-x is a valid HTML attribute.